docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
+ARG PUMPIO__UID=1337
+
+ENV PUMP_LOCATION="/usr/local/pumpio"
```

Comment:
Thinking about this a bit more, it should probably be placed in `/usr/lib/node_modules`. Not entirely sure about this though.